contribute-docs/content-types/_snippets/templates/overview-template.md

```diff
@@ -0,0 +1,115 @@
+---
+description: "A concise summary highlighting what the feature or concept is and why it matters, fit for search results and tooltips."
+applies_to:
+  stack:
+  serverless:
+type: overview
+product:
+---
+
+<!-- Copy and paste this template to get started writing your overview page, deleting the instructions and comments from your final page.
+
+For complete guidance, refer to [the overview guide](https://www.elastic.co/docs/contribute-docs/content-types/overviews).
+
+-->
+
+# [Feature or concept name]
+<!-- REQUIRED
+
+A concise, descriptive name for the feature or concept.
+
+Example: Inference processing
+-->
+
+<!-- REQUIRED
+
+Introduction
```

Comment:
How prescriptive do we want to be? 

I was taught that intros should contain 50 words or fewer in one to two sentences. Best practice is to keep the first sentence to about 25 words to provide just enough information so that users know whether to read on, and so that advanced users can get what they need from the intro and not need to read more. 

Also, never repeat the title.